doc/train/training.md

```
@@ -17,13 +17,27 @@ $ dp --tf train input.json
 ```
 :::
 
+:::{tab-item} TensorFlow 2 {{ tensorflow_icon }}
+
+```bash
+$ dp --tf2 train input.json
```

markdownlint-cli2 (0.23.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)